cmd2.py

```diff
@@ -1148,7 +1173,7 @@ def _cmdloop(self):
             return stop
 
     # noinspection PyUnusedLocal
-    def do_cmdenvironment(self, args):
+    def do_cmdenvironment(self, _):
```

Comment:
I like how using **_** for the argument name makes it nice and explicit and self-documented that the argument is unused.